src/app/accounts/update-contact-alias.ts

```diff
@@ -0,0 +1,31 @@
+import { ValidationError } from "@domain/errors"
+import { AccountsRepository } from "@services/mongoose"
+
+export const updateContactAlias = async ({
+  accountId,
+  username,
+  alias,
+}: {
+  accountId: AccountId
+  username: string
+  alias: string
+}): Promise<AccountContact | ApplicationError> => {
+  const repo = AccountsRepository()
+  const account = await repo.findById(accountId)
+  if (account instanceof Error) {
+    return account
+  }
+
+  const contact = account.contacts.find((contact) => contact.username === username)
+  if (!contact) {
+    return new ValidationError(`User doesn't have contact ${username}`)
```

Comment:
`ValidationError` is just a base class and shouldn't be instantiated directly - please create a new error type.